### PR TITLE
Add smoke test for kibana login.

### PR DESCRIPTION
**Pull request by jmcarp**

### CODE REVIEW

File: smoke-tests-login.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            'username': os.environ['CF_USERNAME'],
+            'password': os.environ['CF_PASSWORD'],
+            'X-Uaa-Csrf': logs.cookies['X-Uaa-Csrf'],
+        }
+    )
+    print(login.url, login.status_code)
+    login.raise_for_status()
+    query = urllib.parse.parse_qs(urllib.parse.urlparse(login.url).query)
+    if 'error' in query:
+        sys.exit(1)
```
Comment (LinuxBozo) [inline smoke-tests-login.py]: To be a test, shouldn't there be at least one assertion of some type?

File: smoke-tests-login.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: python
+    tag: "3.6"
```
Comment (LinuxBozo) [inline smoke-tests-login.yml]: I'd probably use `3.6-alpine` here to keep it small.

File: smoke-tests-login.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import os
+import urllib
+import requests
+
+if __name__ == "__main__":
+    session = requests.Session()
+
+    logs = session.get('https://logs.{}'.format(os.environ['CF_SYSTEM_DOMAIN']))
+    print(logs.url, logs.status_code)
+    assert logs.status_code == 200
```
Comment (cnelson) [inline smoke-tests-login.py]: Shouldn't this be a 302 to UAA / can we assert we are being redirected to login.CF_SYSTEM_DOMAIN?

File: smoke-tests-login.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        data={
+            'username': os.environ['CF_USERNAME'],
+            'password': os.environ['CF_PASSWORD'],
+            'X-Uaa-Csrf': logs.cookies['X-Uaa-Csrf'],
+        }
+    )
+    print(login.url, login.status_code)
+    assert login.status_code == 200
+
+    query = urllib.parse.parse_qs(urllib.parse.urlparse(login.url).query)
```
Comment (cnelson) [inline smoke-tests-login.py]: Probably related to my 302 question above, does this follow redirects?  I'd want to make sure this is exercising the entire oauth flow.

**Comment by jmcarp** [comment]
Requests follows redirects by default. I just threw in assertions that each request winds up at the expected url, but we could also not follow redirects and make assertions about redirects more explicitly. WDYT?

**Comment by cnelson** [comment]
This seems fine to me, no need to make this more complicated than it needs to be.

**Pull request merged**